packaging/setup/ovirt_engine_setup/keycloak/constants.py

```diff
@@ -33,13 +34,47 @@ class Const(object):
     OVIRT_ENGINE_KEYCLOAK_SETUP_PACKAGE_NAME = \
         'ovirt-engine-keycloak'
     KEYCLOAK_MASTER_REALM = 'master'
+    KEYCLOAK_INTERNAL_REALM = 'ovirt-internal'
+    OVIRT_ADMIN_USER = 'admin@ovirt'
```

Comment:
I've just had a discussion with @mwperina about this. We decided to keep admin@ovirt to be able to distinguish from AAA/JDBC  admin@internal.  One of the reasons to have @ovirt  suffix is that on keycloak login page we don't specify a profile or rather realm so we wanted to avoid potential name conflicts with 'admin'  that could in future be imported from sources other than jdbc ie. from ldap (or whatever new  'user backed' our users would think of)